hubstorage/project.py

```diff
@@ -66,7 +67,7 @@ class Jobs(ResourceType):
     resource_type = 'jobs'
 
     def list(self, _key=None, **params):
-        return self.apiget(_key, params=params)
+        raise DeprecatedError('Use list() method from JobQ class instead')
```

Comment:
I don't think we need to invent an exception class for this, I'd use `RuntimeError` instead.

The results of `/jobq/PROJECTID/list` are not exactly the same than `/jobs/PROJECTID/list` but it is possible to obtain similar results, allowing us to log a deprecation warning instead of failure by using: `self.client.get_project(self.projectid).jobq.list(**kwargs)`